### PR TITLE
Enhance/Generalize Monitor

### PR DESCRIPTION
**Pull request by mhmdksh**
This PR is to enhance the gm-monitor and to make it's setup easier and to generalize it's deployment to any app that is hosted on our infra.

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    docker plugin install grafana/loki-docker-driver:2.9.1 --alias loki --grant-all-permissions
+
+    # Step 4: Check if /etc/docker/daemon.json exists and has the required content
+    REQUIRED_CONTENT='{
+    "debug": true,
+    "log-driver": "loki",
+    "log-opts": {
+            "loki-url": "'$LOKI_URL'/loki/api/v1/push",
+            "loki-batch-size": "400",
+            "loki-external-labels": "tenant=services"
```
Comment (geleeroyale) [inline init.sh]: tenant=services is actually a hard-coded value. we should take the opportunity and make it a variable that corresponds to the tenant name set in $INSTANCE (used in promtail config)

File: readme.md [hunk 2]
```diff
@@ -47,47 +54,28 @@ docker-compose down
...
     external:
-      name: giveth-all_giveth
+      name: giveth-all_giveth # Same name of the network that is running in the docker-compose of the app we want to monitor
 ```
 
 2 - check `promtail-config.yml` if you send more than one promtail exporter to your Loki instance:
 
 - the `tenant_id` should be unique
-- the labels for `job name`and `job`should be unique
+- the labels for `job name`and `job`should be unique
```
Comment (geleeroyale) [inline readme.md]: Finally we should add:
3 - Add server IP to logging server LOKI whitelist

**Review by geleeroyale — CHANGES_REQUESTED** [review]
Thanks for this @mhmdksh 

Just two small things to improve (while you are at it)

**Comment by mhmdksh** [comment]
@geleeroyale Please check now

**Review by geleeroyale — APPROVED** [review]
great! thanks a lot!

**Pull request merged**